packages/terra-markdown/src/Markdown.scss

```diff
@@ -1,7 +1,13 @@
-.markdown-body {
-  margin-bottom: 1.25rem;
+:local {
+  .markdown-body {
```

Comment:
Left a comment about not wanting to [hash this class name here](https://github.com/cerner/terra-core/pull/2215/files#r259129492).
We'll want to remove the `:local` select from wrapping these styles.

e.g.
```scss
.markdown-body {
  margin-bottom: 1.25rem;
    
  code {
    background-color: #ebebef;
  }
    
  blockquote {
    color: var(--terra-markdown-blockquote-color, #24292e);
  }
}
```